english/music/add_music.md

```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```

Comment:
I figured it out - let me know if this addition is acceptable!  I added a link to the:
create-your-own-adventure/dance/bake-a-cake/bake-a-cake.md file that has a link to:
create-your-own-adventure/music/add_music.md file, that contains the Queen song "Killer Queen" lyrics, about letting them eat cake, just like Mary Antoinette, and champagne, etc.  Hope it's correctly done this time! ;-)